week3/Exceptions/CopyingFiles.java

```diff
@@ -0,0 +1,32 @@
+package Exceptions;
+
+import java.io.*;
+import java.util.Scanner;
+
+public class CopyingFiles {
+    private static final int BUFFER_SIZE = 4096;
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String inputFile = sc.nextLine();
+        String outputFile = sc.nextLine();
+        File input = new File(inputFile);
+        File output = new File(outputFile);
+

```

Comment:
As per requirement to give  input and output file path direct

Reply:
Fixed